server/issue.go

```diff
@@ -1165,28 +1165,26 @@ func (p *Plugin) fetchConnectedUserFromAccount(account map[string]string, instan
 	return client, connection, nil
 }
 
+func appendAccountInformation(accountID, name string, accountInformation *[]map[string]string) {
+	*accountInformation = append(*accountInformation, map[string]string{
+		jiraUserAccountID: accountID,
+		jiraUserName:      name,
+	})
+}
+
 func (wh *webhook) fetchConnectedUser(p *Plugin, instanceID types.ID) (Client, *Connection, error) {
 	var accountInformation []map[string]string
 
 	if wh.Issue.Fields != nil && wh.Issue.Fields.Creator != nil {
```

Comment:
```suggestion
	if wh.Issue.Fields != nil {
	    if wh.Issue.Fields.Creator != nil {
```
This way, all the three conditions can be present inside the parent `if`.